lib/data/wrapper.js

```diff
@@ -96,7 +96,7 @@ const data = {
             });
     },
 
-    get: (objectGetInfo, log, cb) => {
+    get: (objectGetInfo, log, params, cb) => {
```

Comment:
Oh. A much better idea, and I can get rid of this pr and the Arsenal branch entirely :wink:

Thanks!